src/main/java/com/samcomo/dbz/pin/controller/PinController.java

```diff
@@ -26,57 +25,42 @@ public class PinController {
 
   // Pin 생성
   @PostMapping()
-  public ResponseEntity<CreatePinDto.Response> registerPin(
-      @AuthenticationPrincipal Member member,
-      @ModelAttribute CreatePinDto.Request request,
+  public ResponseEntity<RegisterPinDto.Response> registerPin(
+      @AuthenticationPrincipal MemberDetails details,
+      @ModelAttribute RegisterPinDto.Request request,
       @RequestParam Long reportId
   ){
-    // Authentication 검증
-    String memberEmail = member.getEmail();
 
-    CreatePinDto.Response createPinResponse = pinService.createPin(memberEmail,reportId,request);
-    return ResponseEntity.ok(createPinResponse);
-  }
-
-  // Pin 주소 업데이트 (프론트엔드 - 카카오 API 사용)
-  @PutMapping("/{pinId}/address")
-  public ResponseEntity<UpdatePinAddressDto.Response> updatePinAddress(
-      @AuthenticationPrincipal Member member,
-      @ModelAttribute UpdatePinAddressDto.Request request,
-      @PathVariable Long pinId
-  ){
-    // Authentication 검증
-    String memberEmail = member.getEmail();
+    RegisterPinDto.Response createPinResponse =
+        pinService.registerPin(details.getId(), reportId, request);
```

Comment:
```suggestion
    RegisterPinDto.Response pinResponse =
        pinService.registerPin(details.getId(), reportId, request);
```

변경예정입니다! 한꺼번에 수정할게요/